moto/ecs/models.py

```diff
@@ -868,4 +890,4 @@ def list_task_definition_families(self, family_prefix=None, status=None, max_res
 
 ecs_backends = {}
 for region, ec2_backend in ec2_backends.items():
-    ecs_backends[region] = EC2ContainerServiceBackend()
+    ecs_backends[region] = EC2ContainerServiceBackend(region)
```

Comment:
Whilst your editing this bit, can you change it so it matches how its done [here](https://github.com/spulec/moto/blob/master/moto/batch/models.py#L1042)

Mainly as I know from experience ECS isnt 100% available where EC2 is (currently Mumbai, São Paulo and I'm pretty sure China)

Reply:
Will do!